src/main/java/com/google/codeu/data/Tea.java

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.codeu.data;
+
+// import java.util.UUID;
+
+/** A single message posted by a user. */
+public class Tea {
+    
+    private String userName;
+    private int greenTea;
+    private int whiteTea;
+    private int yellowTea;
+    private int oolongTea;
```

Comment:
Aren't oolong and matcha types of green tea? It sounds like you may need a tree-like structure or hierarchy if you are going to count specific subtypes of tea.

Reply:
For now, I am just going to have Green, White, Black, and Herbal tea. 
I will go back to  implementing the children for the sub teas when I finish the framework for the discussion / threads for the website

Reply:
That sounds like a good idea. Simpler is better.